metadata/src/test/java/org/apache/kafka/controller/ConfigurationControlManagerTest.java

```diff
@@ -72,6 +73,7 @@
             define("ghi", ConfigDef.Type.BOOLEAN, true, ConfigDef.Importance.HIGH, "ghi"));
     }
 
+    private final static ConfigurationValidator NOOP_VALIDATOR = (__, ___) -> { };
```

Comment:
nit: maybe consider moving this so that we can use `ConfigurationValidator.NOOP` or something like that. Then it could also be used in the quorum controller builder then.